### PR TITLE
Split into runtime and devel packages

### PR DESCRIPTION
**Pull request by lemniscati**
- Split into runtime and -devel packages (cf. https://sourceware.org/pipermail/cygwin/2023-January/252787.html)
- Add LICENSE variable
- Rename source archive r56.tar.gz to inih-r56.tar.gz

### CODE REVIEW

**Review by me-and — COMMENTED** [review]
Is this a useful change? Given the whole packages is only ~16KiB, I'd be inclined to leave it as a single package that includes both the binaries and the support files. There are lots of non "-devel" packages at https://cygwin.com/cgi-bin2/package-grep.cgi?grep=usr%2Finclude%2F so I don't think I'd be setting a new precedent there…

**Comment by me-and** [comment]
> Is this a useful change? Given the whole packages is only ~16KiB, I'd be inclined to leave it as a single package that includes both the binaries and the support files. There are lots of non "-devel" packages at https://cygwin.com/cgi-bin2/package-grep.cgi?grep=usr%2Finclude%2F so I don't think I'd be setting a new precedent there…

That was a less useful comment than I thought it would be; I'd thought GitHub was going to comment on the specific commit, not the PR as a whole. I had been _trying_ to comment on [this change](https://github.com/me-and/Cygwin-inih/pull/1/commits/77898c961a508e8a76cb65b93e2cfde9808ffb4e).

File: libinih.cygport [hunk 1]
```diff
@@ -10,7 +10,7 @@ HOMEPAGE='https://github.com/benhoyt/inih'
 LICENSE="BSD-3-Clause"
 # cf. https://www.cygwin.com/packaging-hint-files.html#pvr-src.hint
 #     https://spdx.org/licenses/BSD-3-Clause.html
-SRC_URI="https://github.com/benhoyt/inih/archive/refs/tags/r${PV}.tar.gz"
+SRC_URI="https://github.com/benhoyt/inih/archive/refs/tags/r${PV}/inih-r${PV}.tar.gz"
```
Comment (me-and) [inline libinih.cygport:13]: I had no idea this was possible! That's very good to know, thank you!

**Comment by lemniscati** [comment]
> > Is this a useful change? Given the whole packages is only ~16KiB, I'd be inclined to leave it as a single package that includes both the binaries and the support files. There are lots of non "-devel" packages at https://cygwin.com/cgi-bin2/package-grep.cgi?grep=usr%2Finclude%2F so I don't think I'd be setting a new precedent there…
> 
> That was a less useful comment than I thought it would be; I'd thought GitHub was going to comment on the specific commit, not the PR as a whole. I had been _trying_ to comment on [this change](https://github.com/me-and/Cygwin-inih/pull/1/commits/77898c961a508e8a76cb65b93e2cfde9808ffb4e).

OK.
I just tried to follow a conventional way and Milos' comments.
Please note that Fedora has packages `inih` and `inih-devel` (cf. https://packages.fedoraproject.org/pkgs/inih/)

But, I think discussion of this kind should go on at cygwin-apps ML.

Regards,

**Comment by kmilos** [comment]
> Please note that Fedora has packages `inih` and `inih-devel`

Not only Fedora, [but a bunch of other distros](https://repology.org/project/inih/versions), including Debian etc., and the MSYS2 fork of Cygwin:

https://packages.debian.org/source/bookworm/libinih
https://packages.msys2.org/base/libinih

Anyhow, not a big deal either way...

**Comment by me-and** [comment]
I've merged the changes that aren't splitting the packages, thank you! For splitting the packages, I've asked as part of [my Cygwin Intent To Package email](https://cygwin.com/pipermail/cygwin-apps/2023-January/042477.html), as I definitely don't know what the best practice is for Cygwin, or even if there is one…

**Pull request closed without merging**